lib/internal/crypto/random.js

```diff
@@ -119,6 +120,51 @@ function randomFill(buf, offset, size, cb) {
   _randomBytes(buf, offset, size, wrap);
 }
 
+// Largest integer we can read from a buffer.
+// e.g.: Buffer.from("ff".repeat(6), "hex").readUIntBE(0, 6);
+const RAND_MAX = 281474976710655;
+
+function randomInt(min, max, cb) {
+  if (typeof cb !== 'function') {
+    throw new ERR_INVALID_CALLBACK(cb);
+  }
+  if (!Number.isInteger(min)) {
+    throw new ERR_INVALID_ARG_TYPE('min', 'integer', min);
+  }
+  if (!Number.isInteger(max)) {
+    throw new ERR_INVALID_ARG_TYPE('max', 'integer', min);
+  }
```

Comment:
It’s not super important, but these days we usually prefer cached built-ins, i.e. `NumberIsInteger()` instead of `Number.isInteger()` so that we’re not affected when userland code decides to override that function.

You can import that cached builtin by adding `NumberIsInteger` to the list above.

Reply:
Thanks, fixed.